packages/cli/test/integration.js

```diff
@@ -45,6 +45,12 @@ function fixture(name) {
   return directory;
 }
 
+const URL_REGEX = /(https?:\/\/[^ ]*)/;
+function extractUrl(input) {
+  let match = input.match(URL_REGEX);
+  return match?.[1];
+}
```

Comment:
This allows the tests that rely on stdout containing a URL to still pass when you are trying to add debugging logs for other tests. When these tests fail because of those extra logs, it's essentially a false negative that wastes time.

If we want tests to fail when we introduce superfluous output in general, we should tackle that as an explicit effort.

Reply:
I'll remove these and we can look at this more later.